src/pages/ResultsPage/Map.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 
 import Script from 'react-load-script';
 
-const Map = ({ waypoints }) => {
+const Map = ({ waypoints, polyline }) => {
```

Comment:
Throw this into proptypes below. Honestly, it doesn't matter as much since we're destructuring props here (`({prop1, prop2})` vs `(props)`) but just for consistency. 

Just do `polyline: PropTypes.array`